src/elements/hx-search/index.js

```diff
@@ -1,7 +1,7 @@
 import { HXElement } from '../../interfaces/HXElement/index.js';
 
 import shadowMarkup from './_shadow.html';
-import shadowStyles from './_shadow.less';
+import shadowStyles from './_shadow.scss';
```

Comment:
@CITguy, you're making changes to `<hx-search>` Shadow DOM LESS -> SCSS.

Noting here for PR #592 `<hx-search>` refactor.